.github/workflows/release.yml

```diff
@@ -33,11 +40,19 @@ jobs:
         run: cargo +stable install cargo-make cargo-near
       - name: Compile contract
         run: cargo make build
+
+      - name: Generate launchpad types
+        run: |
+          chmod +x scripts/gen-launchpad-types.sh
+          ./scripts/gen-launchpad-types.sh
+
       - name: Upload artifacts
         uses: actions/upload-artifact@v4
         with:
           name: contracts
-          path: res/aurora-launchpad-factory.wasm
+          path: |
+            res/aurora-launchpad-factory.wasm
+            res/schema.ts
```

Comment:
I think the better approach is to publish the TS file as npm package to nmpjs.com